apps/prairielearn/src/ee/lib/validateHTML.ts

```diff
@@ -281,7 +322,7 @@ function checkIntegerInput(ast: DocumentFragment | ChildNode): ValidationResult
           break;
         // string inputs are valid as strings, and these don't affect other tags, so no validation required
         case 'correct-answer':
-          if (val.match(mustacheTemplateExtractorRegex)) {
+          if (val.match(mustacheTemplateRegex)) {
```

Comment:
I think the only difference here is that the previous one would reject something like `correct-answer="foo {{ bar }}"`, whereas this would only reject `correct-answer="{{ bar }}"`. IMO this is fine, but we can make it looser if we want.

Reply:
I doubt that this is much of an issue anyways. See https://github.com/PrairieLearn/PrairieLearn/pull/11955#discussion_r2085209380.